ansible/all-in-one/roles/xsnippet/templates/xsnippet-webserver-spa.conf.j2

```diff
@@ -38,8 +36,8 @@ server {
     gzip_types text/plain text/css application/json application/x-javascript text/javascript application/javascript image/svg+xml;
     gzip_min_length 1024;
 
-    location ~ ^/(\d+)/raw/?$ {
-        proxy_pass http://xsnippet_web_backend/snippets/$1/raw;
+    location ~ ^/([a-zA-Z0-9]+)/raw/?$ {
```

Comment:
Just to double check: snippet ID can't have `_` in it?

Reply:
I've copied the regex from our API tests :) Slugs can only contain alphanumeric characters.